src/app/components/datepicker/index.html

```diff
@@ -6,6 +6,10 @@
 
   <sky-demo-page-summary>
     The datepicker module contains directives to let users select dates.  It is a SKY UX-themed replacement for the HTML <sky-code>input</sky-code> element with <sky-code>type="date"</sky-code>. The value that users select is driven through the <sky-code>ngModel</sky-code> attribute specified on the <sky-code>input</sky-code> element.
+
+    <p>
+      To use the datepicker module, you must also install <sky-code>SkyPopoverModule</sky-code> as a dependency to your project. For more information on <sky-code>SkyPopoverModule</sky-code>, see <a routerLink="/components/popover">Popover module</a>.
```

Comment:
I'd delete "also" in the first sentence and "more" in the second sentence.

I also think this should be in the Installation section, but that would take some work. Given that we are reworking the Installation section a bit, I think we should leave this where it is for now and move it to the Installation section later.